src/Scores.cpp

```diff
@@ -386,8 +385,18 @@ void Scores::createXvalSetsBySpectrum(std::vector<Scores>& train,
   for (unsigned int i = 0; i < xval_fold; ++i) {
     train[i].recalculateSizes();
     test[i].recalculateSizes();
+
+    // show the size of each fold
+    /*
+     * The intuition of why the splits are approximately balanced:
+     * Imagine you have a random generator which generates random integers.
+     * Now you randomly assigned the generated numbers into k fold by MOD k,
+     * As long as you generate sufficient numbers, it's very likely to get approximately even split.
+     */
+    std::cerr << "Fold=" << i << "\tTrain size=" << train[i].size() << "\tTest size=" << test[i].size() << std::endl;
```

Comment:
I understand that it intuitively works, but the question is if it works in practice for different sets of inputs. 

Also, I guess this code was only used for testing, so please remove this entire block from the pull request to reduce code pollution. (If you want to add proper log messages, enclose them in `if (VERB > x) { ... }` blocks.)